custom/opm/reports.py

```diff
@@ -500,7 +500,7 @@ def join_rows(self, rows):
         if len(rows) == 1:
             return rows[0]
         def zip_fn((i, values)):
-            if i == self.column_index('num_children'):
+            if i == self.column_index('num_children') or i == self.column_index('year_end_bonus_cash'):
```

Comment:
got it, thanks. might be more readable with a comment or by splitting out the conditions but doesn't need to block.